### ISSUE TITLE
Hi i have a problem with the toast feature

### ISSUE THREAD

**Issue by YassiGame** [hotpocket184/skript-advancements#7]
Hi evryone, i have a issue with the tost feature, i have the last version of the addon and i have a 1.18.2 server... Please help 😅
I wrote this script, I only used the example from skripthub docs:
```
command /test:
	trigger:
		send toast to player named "You're So Cool!" with icon book with frame goal
		send "this is a tost"
```
I tested it but nothing works I have purpur as a server (fork of paper), 

```
[13:35:21 INFO]: YassiGame issued server command: /test
[13:35:21 INFO]: Loaded 7 recipes
```
```
[13:16:44 INFO]: YassiGame issued server command: /test
[13:16:44 INFO]: [NBTAPI] [NBTAPI] Found Spigot: v1_18_R2! Trying to find NMS support
[13:16:44 INFO]: [NBTAPI] [NBTAPI] NMS support 'MC1_18_R2' loaded!
[13:16:44 INFO]: [NBTAPI] [NBTAPI] Found Gson: class com.google.gson.Gson
[13:16:44 INFO]: [NBTAPI] [NBTAPI] The NBT-API seems to be up-to-date!
[13:16:46 INFO]: Loaded 7 recipes
```

I looked for the closed issues to see if there is a solution to this problem, I found this issue #6 
He has the same problems as me

**Comment by hotpocket184** [comment]
If you're using version 1.3.4 of skript-advancements then it shouldn't reload recipes or anything like that. Check again to see if you have the most recent version.

**Comment by hotpocket184** [comment]
No problem. People still have 1.3.2 mainly because it's on spigot and I never added the released to GitHub until 1.3.3. Luckily, there is now an update checker to help prevent this sort of stuff. 😉
